scripts/fallow-badge.mjs

```
@@ -13,40 +13,21 @@ if (!inputPath || !badgePath || !metricsPath) {
-  unused_enum_members: 0.5,
-  unused_class_members: 0.5,
-  type_only_dependencies: 0.5,
-  test_only_dependencies: 0.5,
-};
+const averageMaintainability = Number(healthSummary?.average_maintainability);
 
-const weightedIssues = Object.entries(weights).reduce((total, [key, weight]) => {
-  return total + (Number(summary[key] ?? 0) * weight);
-}, 0);
+if (!Number.isFinite(averageMaintainability)) {
+  console.error("Fallow native maintainability not found in report JSON");
```

ESLint

[error] 'console' is not defined.

(no-undef)